app/notifications/push/index.ts

```diff
@@ -0,0 +1,76 @@
+import EJSON from 'ejson';
+
+import store from '../../lib/createStore';
+import { deepLinkingOpen } from '../../actions/deepLinking';
+import { isFDroidBuild } from '../../constants/environment';
+import PushNotification from './push';
+
+export interface INotification {
+	message: string;
+	style: string;
+	ejson: string;
```

Comment:
`ejson: IEjson;` maybe ?

Reply:
Not because it's a string anyway, it only changes into json using the lib.